docs/source/services/timelock_service/product-changes.rst

```diff
@@ -0,0 +1,20 @@
+.. _product-changes:
+
+Developing a product to use the timelock service
+================================================
+ .. note::
+    We recommend only product instances deployed against Cassandra KVS to use the timelock server.
```

Comment:
Why is this? Because of the lack of automated migration?

nit: the "to" in this sentence sounds a bit odd